internal/driver/checkstatuses.go

```diff
@@ -18,17 +19,24 @@ import (
 // checkStatuses loops through all registered devices and tries to determine the most accurate connection state
 func (d *Driver) checkStatuses() {
 	d.lc.Debug("checkStatuses has been called")
+	wg := sync.WaitGroup{}
 	for _, device := range service.RunningService().Devices() {
 		if device.Name == d.serviceName { // skip control plane device
 			continue
 		}
 
-		status := d.testConnectionMethods(device)
+		device := device // save the device value within the closure
```

Comment:
please use a different variable name for the async function to improve the readability and future maintenance

Reply:
@cloudxxx8 I actually find that the re-use of the name makes it more maintainable, as it takes the outer `device` var out of scope, whereas with a different name, 2 variables are in scope and the developer could use the wrong one.

Having said that, I think I should move this line to first line of the for loop to prevent it from being lost/moved, etc.